iidm/iidm-impl/src/test/java/com/powsybl/iidm/network/impl/TieLineTest.java

```diff
@@ -27,6 +27,7 @@
  * @author Luma Zamarreño <zamarrenolm at aia.es>
  * @author José Antonio Marqués <marquesja at aia.es>
  */
+@Ignore
```

Comment:
@marqueslanauja Can you fix these tests so they can not be ignored anymore?

Reply:
Done. I Fixed them by increasing the tolerance with the error made when using the danlingLine model instead of the initial line model used to define the case.